addons/ondevice-actions/package.json

```diff
@@ -26,13 +26,13 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
-    "@storybook/addons": "6.0.28",
```

Comment:
Doesn't it make sense to be a bit more lenient with these versions for users (like me) that use storybook for multiple platforms? We can still lock it down to a certain/single version in the lockfile for this repo.

Reply:
For example changing the addons dependencies to peer dependencies?

Reply:
Not necessarily, but now you're essentially forcing consumers to use one single version of the `@storybook/*` packages, and it's not unlikely that they might have other dependencies wanting the same package and will thus end up with multiple versions and potentially incompatibilities.

Reply:
I don't have a particular reason to force a version. However I would say that with 6.0 there are going to be things that only work with 6.0+. 

However I'm open to suggestions on how we can improve this experience for your use case. Perhaps you could make an issue that describes the use case and has some potential solutions? Then we can discuss it further and include it in the 6.0 project board if it makes sense to do so.

Reply:
What do you think about setting it to `^6`?

Reply:
I understand that the main storybook repo/package needs to keep their versions strict, but since this is maintained as an separate workspace/package with its own release cycle it shouldn't be strict IMHO.

Reply:
Something like this?
```
  "dependencies": {
    "core-js": "^3.0.1",
    "fast-deep-equal": "^2.0.1"
  },
  "devDependencies": {
   "@storybook/addons": "6.3.0-alpha.33",
    "@storybook/core-events": "6.3.0-alpha.33",
    "@storybook/addon-actions": "6.3.0-alpha.33"
  },
  "peerDependencies": {
   "@storybook/addons": "^6",
    "@storybook/core-events": "^6",
    "@storybook/addon-actions": "^6",
    "react": "*",
    "react-native": "*"
  },
```

Reply:
Also in this PR I'm using the alpha version because thats where the new sb-modern field was introduced, so once 6.3 gets merged it would be possible to do ^6.